src/native/fs/fs_napi.cpp

```diff
@@ -279,6 +337,47 @@ struct Stat : public FSWorker
     }
 };
 
+
+/**
+ * temp_file support by spectrumscale
+ */
+struct tempFile : public FSWorker
```

Comment:
Can't this be part of FileWrap or DirWrap? As we always do with operation interacting with file descriptors

Reply:
it can but I think a function to get the fd will be useful in the future since most gpfs ops take a fd

Reply:
will we be using an fd for files not needed to being opened and closed. If we don't then I think we should be using the wrapper, so it will help us to remember to close the file. @guymguym what do you think?

Reply:
ok, I will move it.